site/zenodo_rdm/communities_ui/views/communities.py

```diff
@@ -58,30 +59,43 @@ def communities_home(pid_value, community, community_ui):
         return redirect(url)
 
     if theme_enabled:
+        metrics = current_cache.get(
+            f"THEMED_COMM_METRICS_{community._record.theme['brand']}"
+        )
```

Comment:
```suggestion
        metrics = current_cache.get(
            f"themed_comm_metrics:{community.id}"
        )
```

- Lowercase and use `:` for namespacing (kind of a Redis standard)
- We should use the community ID, since we might in the future apply the same branding on multiple communities